common/plane.h

```diff
@@ -0,0 +1,26 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2021 Intel Corporation. All Rights Reserved.
```

Comment:
Can you please update the copyrights date to 2023?

Reply:
Yep, will do. Apologies, but my git is still a bit rusty and I think I messed up my branch the my local version. I also think I branched off the wrong version to begin with which is why auto-merge isn't possible. Gone ahead and re-implemented my change in https://github.com/IntelRealSense/librealsense/pull/11404. Sorry again for the churn.